modules/web3/ens.ts

```diff
@@ -11,3 +11,13 @@ export async function getENS(address: string): Promise<string> {
   const resolver = await ens.reverse(address);
   return await resolver.name();
 }
+
+export async function resolveENS(ensName: string): Promise<string> {
+  const maker = await getMaker();
+
+  const provider = maker.service('web3')._web3.currentProvider;
+
+  const ens = new ENS(provider);
+
+  return await ens.resolver(ensName).addr();
```

Comment:
FYI ethers has an ENS resolver we could use: https://docs.ethers.io/v5/api/providers/provider/#Provider--ens-methods

it seems that the `ethereum-ens` package hasn't been updated in a while